modules/pushapk_scriptworker/files/requirements.txt

```diff
@@ -39,7 +39,7 @@ kiwisolver==1.0.1
 lxml==4.3.0
 matplotlib==3.0.2
 mohawk==0.3.4
-mozapkpublisher==0.10.1
+mozapkpublisher==0.11.0
```

Comment:
Sorry, this line needs to be rebased because I quickly landed. https://github.com/mozilla-releng/build-puppet/pull/362